docs/user/vulnerabilities/vulnerability/product_status-usage.en.md

```diff
@@ -1,27 +1,51 @@
 # Product Status - Usage
 
-* The same product ID must not be member of contradicting product status groups.
-  I.e. a product listed in the group of _affected_ products may not be listed again in the group of _not affected_ products.
-  The contradicting groups are
+The product status provides information on the status products in the `product_tree` are in.
 
-  * affected
+Generally, vulnerable (`known_affected`) and fixed (`fixes`) versions of products should be given.
 
-    * `product_status/first_affected[]`
-    * `product_status/known_affected[]`
-    * `product_status/last_affected[]`
+The products are listed by their ID in different lists of product status groups.
+The same product ID must not be member of contradicting product status groups.
+I.e. a product listed in the group of _affected_ products may not be listed again in the group of _not affected_ products.
 
-  * not affected
+The contradicting groups are
 
-    * `product_status/known_not_affected[]`
+* affected
 
-  * fixed
+  * `product_status/first_affected[]`
+  * `product_status/known_affected[]`
+  * `product_status/last_affected[]`
 
-    * `product_status/first_fixed[]`
-    * `product_status/fixed[]`
+* not affected
 
-  * under investigation:
+  * `product_status/known_not_affected[]`
 
-    * `product_status/under_investigation[]`
+* fixed
 
-  > Note: An issuer might recommend (product_status/recommended) a product version from any group - also from the
-  > affected group, i.e. if it was discovered that fixed versions introduce a more severe vulnerability.
+  * `product_status/first_fixed[]`
+  * `product_status/fixed[]`
+
+* under investigation:
+
+  * `product_status/under_investigation[]`
+
+> Note: An issuer might recommend (product_status/recommended) a product version from any group - also from the
+> affected group, i.e. if it was discovered that fixed versions introduce a more severe vulnerability.
+
+Each product listed in the affected group should be listed in the remediations (`/vulnerabilities[]/remediations`).
+
+For each product listed in the not affected group, the reason why it is not affected should be given.
+This reason should be listed in the threats (`/vulnerabilities[]/threats`) with category `impact` (Impact Statement) or as machine-readable Impact Statement in the flags (`/vulnerabilities[]/flags`).
+It is recommended to use both.
+
+The status `known_not_affected` is used in the context of security advisories to exclude individual products. This is mainly used for products that are likely to be affected.
+To define general non-affectedness, the VEX profile should be used.
+
+An issuer might recommend (`product_status/recommended`) a product version from any group - also from the affected group, i.e. if it was discovered that fixed versions introduce a more severe vulnerability.
```

Comment:
That is also a comment above. Please de-duplicate.

Reply:
removed above